content/en/docs/armory-admin/policy-engine-enable/_index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Policy Engine for Armory Enterprise
+linkTitle: Policy Engine
+description: >
+  Enable the Policy Engine to enforce policies on your Armory Enterprise instance. Policies can help you make sure that best practices are followed by preventing pipelines from being saved or running if they do not meet requirements that you outline. This page includes how to deploy and configure an OPA sever, which the Policy Engine requires.
+aliases:
+  - /docs/plugin-guide/plugin-policy-engine/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+The Armory Policy Engin is a proprietary feature for Armory Enterprise that is designed to allow you more complete control over the software delivery process. The Policy Engine provides the hooks necessary to perform  extensive verification of pipelines and processes in Armory Enterprise. The Policy Engine uses the [Open Policy Agent](https://www.openpolicyagent.org/)(OPA) and input style documents to perform validations on the following:
+
+* **Save time validation** - Validate pipelines as they're created/modified. This validation operates on all pipelines using a fail closed model. This means that if you have the Policy Engine enabled but no policies configured, the Policy Engine prevents you from creating or updating any pipeline.
+* **Runtime validation** - Validate deployments as a pipeline is executing. This validation only operates on tasks that you have explicitly created policies for. Tasks with no policies are not validated.
+* **Entitlements using API Authorization** - Control what actions can be taken based on a user's role. (Requires the Policy Engine plugin.)
+
+The Policy Engine exists as a plugin, which is its newer iteration, and as an extension of Armory Enterprise. The plugin has additional features that are not present in the extension. If you are getting started with the Policy Engine, Armory recommends using the plugin version of the Policy Engine. If you want to migrate from the extension to the plugin, see [Migrating to the Policy Engine Plugin](#migrating-to-the-policy-engine-plugin).
+
+For information about how to use the Policy Engine, see [Using the Policy Engine]({{< ref "policy-engine-use" >}}).
+
+## Requirements for using the Policy Engine
+
+Make sure you can meet the following version requirements for the Policy Engine:
+
+* OPA versions 0.12.x or later. Specifically, the OPA v1 API must be available.
```

Comment:
.12 is what we've had for awhile. We did update the config map that deploys an OPA server to a more recent version. Should this min version track whatever version we use in the configmap

Reply:
From Slack: We're keeping .12 as a min version, but we should keep the config map updated.